src/templates/exportService.hbs

```diff
@@ -18,10 +16,10 @@ import { ${{{this}}} } from '../schemas/${{{this}}}';
 {{/if}}
 {{/if}}
 
-import { globalOptionsAtom, request as __request } from '../core/request';
-import type { ResultState, IActionCallback } from '../core/request';
-import { BaseApiRequestOptions, ApiRequestOptions } from '../core/ApiRequestOptions';
-import { ApiResult } from '../core/ApiResult';
+import { request as __request } from '../core/request';
+import { BaseApiRequestOptions, ApiRequestOptions } from 'api/core/ApiRequestOptions';
```

Comment:
Users not configured absolute import can't import modules in this way, please fix.